docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -21,7 +21,7 @@ $ curl -X GET api/task_manager/_health
 --------------------------------------------------
 // KIBANA
 
-Monitoring the `_health` endpoint of each {kib} instance in the cluster is the recommended method of ensuring confidence in mission critical services such as Alerting and Actions.
+Monitoring the `_health` endpoint of each {kib} instance in the cluster is the recommended method of ensuring confidence in mission critical services such as Alerting, Actions and Reporting.
```

Comment:
This looks unrelated to the original issue - is this a known issue that the docs aren't properly including reporting?

Reply:
I noticed that sometimes it read `Alerting and Actions` and sometimes it read `Alerting and Reporting` and sometimes it read `Alerting, Actions and Reporting` so I tried to use the same language. There's no issue filed for it I don't think...

Reply:
What does Alerting, Actions and Reporting refer to?  Maybe we should open a separate issue to address this.

Reply:
What is Actions?

Reply:
@gchaps Actions are another type of task that's handled by task manager. I am happy to back out these changes and open a separate issue to address this.

Reply:
Created new issue: https://github.com/elastic/kibana/issues/119409